magda-typescript-common/src/test/mockAuthApiHost.ts

```diff
@@ -0,0 +1,180 @@
+import * as nock from "nock";
```

Comment:
Appreciate the effort here, but I would've already been more than happy if the tests for `ApiClient` just checked that each method made the right HTTP calls!

What you've done here is great but a bit excessive :)

Reply:
;-) I have to say I regretted in the half way... but thought better to close it up.

Reply:
I'm very impressed with the thoroughness though 😄